Sources/TokamakStaticHTML/StaticHTMLFiberRenderer.swift

```diff
@@ -188,16 +207,42 @@ public struct StaticHTMLFiberRenderer: FiberRenderer {
       case let .update(previous, newContent, _):
         previous.update(with: newContent)
       case let .layout(element, data):
-        print("Received layout message \(data) for \(element)")
+        element.content.attributes["style", default: ""] += """
+        position: absolute;
+        left: \(data.origin.x)px;
+        top: \(data.origin.y)px;
+        width: \(data.dimensions.width)px;
+        height: \(data.dimensions.height)px;
+        """
       }
     }
   }
 
   public func measureText(
     _ text: Text,
-    proposedSize: CGSize,
+    proposal: ProposedViewSize,
     in environment: EnvironmentValues
   ) -> CGSize {
     .zero
   }
+
+  public func render<A: App>(_ app: A) -> String {
+    _ = FiberReconciler(self, app)
+    return """
+    <!doctype html>
+    <html>
+    \(rootElement.description)
+    </html>
+    """
+  }
+
+  public func render<V: View>(_ view: V) -> String {
+    _ = FiberReconciler(self, view)
+    return """
+    <!doctype html>
+    <html>
+    \(rootElement.description)
+    </html>
+    """
+  }
```

Comment:
will the presence of `doctype` here make https://github.com/TokamakUI/Tokamak/pull/489 obsolete?

Reply:
I believe that PR deals with the stack reconciler `StaticHTMLRenderer`, not this fiber reconciler one.
Although this could be enhanced to support meta tags in the head and such once preferences are supported in the fiber reconciler.